e2e/benchmarks/local-benchmark/index.html

```diff
@@ -797,7 +850,20 @@ <h2>TensorFlow.js Model Benchmark</h2>
       updateGUIFromURLState();
     }
 
-    onPageLoad();
+    /**
+     * Check if specific backend is supported in current browser.
+     * @param {string} backendName
+     */
+    function isBackendSupported(backendName) {
+      if (backendName === 'webgpu' && typeof navigator !== 'undefined' &&
```

Comment:
How about typeof navigator == 'undefined'?

Reply:
Browser will always has navigator, and it's a read-only object, so remove condition above.